kong/api/routes/health.lua

```diff
@@ -11,6 +12,7 @@ local knode  = (kong and kong.node) and kong.node or
 local select = select
 local tonumber = tonumber
 local kong = kong
+local dbless = kong.configuration.database == "off"
```

Comment:
We must expand this to include role=data_plane instances as well.

Reply:
No added with 1acca49962a8c1e4fbc1f598f5bf892f97ec1b5e :+1: